src/app/(Authentication)/Login/Password/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { router, useLocalSearchParams } from 'expo-router';
+import React, { useState } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import AuthInput from '../../../../Components/AuthInput/AuthInput';
+import { passwordExists } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  const [password, setPassword] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+  const { email } = useLocalSearchParams() as unknown as { email: string };
+  const [displayPassword, setDisplayPassword] = useState(false);
```

Comment:
We can do the same thing here. Add a `placeholder` state variable and pass it down to `AuthInput`